src/constraints.jl

```diff
@@ -8,10 +8,26 @@ is, `copy_to(model, src)` does not throw [`UnsupportedConstraint`](@ref) when
 `src` contains `F`-in-`S` constraints. If `F`-in-`S` constraints are only not
 supported in specific circumstances, e.g. `F`-in-`S` constraints cannot be
 combined with another type of constraint, it should still return `true`.
-"""
+
+    supports_constraint(model::ModelLike, ::Type{VectorOfVariables}, ::Type{Reals})::Bool
+
+Return a `Bool` indicating whether `model` supports free variables. That is,
+`copy_to(model, src)` does error when `src` contains variables that are not
```

Comment:
The double negative "does not error when ... contains variables that are not ..." is pretty confusing. Could you provide a concrete example? E.g., if a solver requires all variables to be nonnegative.